testing/marketplace/deployer_envsubst_base/standard/test

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -eox pipefail
+
+test_directory=$(mktemp --dry-run)
+cp -r $(dirname "$0") "$test_directory"
+cd "$test_directory"
+
+image="$REGISTRY/testing/deployer_envsubst/1:$MARKETPLACE_TOOLS_TAG-$TEST_ID"
```

Comment:
s/1/standard/. Or some bash construct to extract the last portion of the directory path

Reply:
Done!